### ISSUE TITLE
Item crashing after update

### ISSUE THREAD

**Issue by mi747** [Ben216k/Patched-Sur#40]
**Describe the bug**
A clear and concise description of what the bug is.  After update to Big Sur 11.6.2 , Weatherbug crashes 

**To Reproduce**
1.  Click Weatherbug. 
2. Click on temperature in menu bar  
3. Weatherbug displays information, but when it displays map it crashes
4. Gives no error 

**Expected behavior**
Should remain open after map is displayed, after clicking on map, it should open Firefox and display larger view of map 
**Screenshots**

**Desktop (please complete the following information):**
 - Upgrading from: Big Sur 11.5.2
- Upgrading to: Big Sur 11.6.2
 - Mac Model: Mac mini Late 2012
 - Version [e.g. v0.0.4]

**Additional context**
Add any other context about the problem here.  I've tried to uninstall and reinstall Weatherbug it did not fix problem.

If there are any red boxes, click on them to copy the logs.

**Comment by Ben216k** [comment]
I would suggest talking to the developer of the app.

**Comment by mi747** [comment]
I got your email this morning thanks for the reply.  After I upgraded to Big Sur 11.6.2 it works perfect except it will not let me display maps on any search engine, Safari, Firefox, Microsoft edge, Google or Weather bug.  Is there a way to correct this.  I first noticed it using Weather bug so that started my request for help, but it turns out all maps will not work, on Safari, Firefox and Weather bug  it will crash the whole page, on Google and Microsoft edge it just will not display the maps.
